src/assets/styles/components/_tooltip.scss

```diff
@@ -0,0 +1,30 @@
+.tooltip-class {
+    z-index: 1000;
+}
+
+.tooltip-inner {
+    background-color: #f3f4f6;
+    position: relative;
+    font-size: 12px;
+    border: 1px solid #ced4da;
```

Comment:
I suggest adding a variable for border color,
there are three places here with this color, and I think it should be the same all the time.
For example `$border-color: #ced4da;` and replace border-color with this variable in all 3 places

Reply:
Ok. fixed